antenna/lib/ooid.py

```diff
@@ -0,0 +1,112 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This is based on socorrolib's socorrolib/lib/ooid.py module.
+
+# OOID is "Our opaque ID"
+import datetime as dt
```

Comment:
It's better for project archaeology and history to keep conversations in one place or another rather than have conversations diverge.